### PR TITLE
Merging display-cards to main

### PR DESCRIPTION
**Pull request by Celinarabe**
Code review

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -69,9 +69,8 @@ function App() {
           highScore={highScore}
           resetScore={resetScore}
         />
-        <p>{cards[0].name}</p>
 
-        <Cards addScore={addScore} handleClick={(id) => handleClick(id)} />
+        <Cards cardDeck={cards}addScore={addScore} handleClick={(id) => handleClick(id)} />
```
Comment (gtaylor5) [inline src/components/App.js]: For readability add a space before addScore.
```
<Cards cardDeck={cards} addScore={addScore} ... />
```

File: src/components/Cards.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ export default function Cards(props) {
   return (
     <div>
       <ul>
-        {cactusArr.map((value, index) => {
-          return <img key={index} src={value} alt={`cactus${index}`} onClick={() => props.handleClick(index)} />;
+      {props.cardDeck.map((value, index) => {
+          return <img key={value.id} src={value.name} alt={`cactus${value.id}`} onClick={() => props.handleClick(value.id)} />;
         })}
```
Comment (gtaylor5) [inline src/components/Cards.js]: Let's not do this in-line. We can pull this out into a constant function. So:
```
const generateCards = () => {
  return props.cardDeck.map((value, index) => {
    return ...
  }
}
```

Then in your return

```
return (
  <div>
    <ul>
      {generateCards()}
    </ul>
</div>);
```

By doing this, we can make the component more testable.

File: src/helpers/initializeDeck.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ import cactusArr from "../img/";
 //card object: id and name (full src path)
 
 export default function initializeDeck() {
-  console.log("this is being ran");
   let id = 0;
   let cardDeck = [];
```
Comment (gtaylor5) [inline src/helpers/initializeDeck.js]: I don't think this needs to be a `reduce`. I think you can just use `map`.

```
return cactusArr.map(name => {
  let card = { id: ++id, name };
  return card;
});
```

**Review by gtaylor5 — COMMENTED** [review]
Overall, this looks great! Seems like you have a really good grasp of the concepts already :)

File: src/components/App.js [hunk 4]
```diff
@@ -1,54 +1,37 @@
...
+import { Container } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
 
 import initializeDeck from "../helpers/initializeDeck";
+import shuffleDeck from "../helpers/shuffleDeck";
 
 function App() {
-  //returns current state and async function to update state
-  const [cards, setCards] = useState([{}]);
+  //useState returns current state and async function to update state
```
Comment (gtaylor5) [inline src/components/App.js:12]: Just want to clarify this:

`initializeDeck` is not an async function. It's actually synchronous.

File: src/components/Cards.js [hunk 5]
```diff
@@ -1,18 +1,23 @@
 import React from "react";
 import "../css/Cards.css";
-import cactusArr from "../img/";
 
 export default function Cards(props) {
```
Comment (gtaylor5) [inline src/components/Cards.js]: nit: 

Let's try not to export the function directly in the future. Also, prefer arrow function syntax. 

```
const Cards = (props) => {
  ...
}
```

export default Cards;

**Review by gtaylor5 — APPROVED** [review]
Looks great! Merge this in! 

A word on "nit" comments. It's up to you to to decide whether or not you want to actually make the change. Nits usually refer to the user's preference.

**Review by gtaylor5 — APPROVED** [review]

File: src/components/Cards.js (hunk 5 above)
Comment (Celinarabe) [reply src/components/Cards.js]: converted functions in src/components and src/helpers to arrow function syntax and changed to indirect exports

**Pull request merged**